node/package.json

```diff
@@ -1,7 +1,8 @@
 {
-  "name": "node-starter",
+  "name": "node-demo",
   "version": "0.0.0",
+  "private": true,
   "scripts": {
-    "test": "echo \"Error: no test specified\" && exit 1"
+    "start": "node index.js"
```

Comment:
Instead of telling users to run `node index.js` in the Terminal as a comment in `index.js`, we can just run it on startup.